DBZ-6605 Fix DataCollections for snapshot completion notification

bdbene opened the pull request:
https://issues.redhat.com/browse/DBZ-6605

This is a fix to see only the table name for which the table scan has completed. The notification should be per table so that the it can be acted on correctly.

jpechane commented:
@mfvitale Could you please review the PR? The proposal seems logical to me.

mfvitale commented on debezium-core/src/main/java/io/debezium/pipeline/notification/IncrementalSnapshotNotificationService.java:
```diff
@@ -99,9 +99,7 @@ public <T extends DataCollectionId> void notifyAborted(IncrementalSnapshotContex
 
-        String dataCollections = incrementalSnapshotContext.getDataCollections().stream().map(DataCollection::getId)
-                .map(DataCollectionId::identifier)
-                .collect(Collectors.joining(LIST_DELIMITER));
```
What do you think to add a new field (maybe `SCANNED_COLLECTION`) as for in progress notification (`CURRENT_COLLECTION_IN_PROGRESS`) and leave `DATA_COLLECTIONS` unchanged?

jpechane replied:
Sounds good!

bdbene replied:
Thanks for the quick feedback! I'll make the changes.

mfvitale commented:
@bdbene The proposal is good! Please

- [ ] address the suggested change
- [ ] change documentation accordingly (https://github.com/debezium/debezium/blob/fa78e777977e49c8f54260609099b3ccb1ed352e/documentation/modules/ROOT/pages/configuration/notification.adoc?plain=1#L151)

bdbene commented:
I've made the changes; added a new field to indicate for which table the snapshot has completed. And I've updated the docs to indicate that the existing field is a list of the remaining tables, and the new field has the completed table.

mfvitale commented:
@bdbene good work! Thank you.
